src/app/tags/components/KernelOptionsField/KernelOptionsField.test.tsx

```diff
@@ -101,23 +111,45 @@ it("displays a deployed machines message when updating a tag", async () => {
   ).toBeInTheDocument();
 });
 
-it("displays a deployed machines message when passed machines", async () => {
-  const machines = [
-    machineFactory({
-      status: NodeStatus.DEPLOYED,
-      tags: [1],
-    }),
-  ];
+it("displays a deployed machines message when passed deployedMachinesCount", async () => {
+  const store = mockStore(state);
+  render(
+    <Provider store={store}>
+      <MemoryRouter>
+        <Formik initialValues={{}} onSubmit={jest.fn()}>
+          <KernelOptionsField deployedMachinesCount={1} />
+        </Formik>
+      </MemoryRouter>
+    </Provider>
+  );
```

Comment:
This could use renderWithMockStore or renderWithBrowserRouter

Reply:
Sure, good suggestion!